code/game/objects/items/robot/robot_upgrades.dm

```diff
@@ -400,38 +404,37 @@
 
 	if(cyborg && (cyborg.stat != DEAD) && on)
 		if(!cyborg.cell)
-			to_chat(cyborg, "<span class='warning'>Self-repair module deactivated. Please, insert the power cell.</span>")
+			to_chat(cyborg, "<span class='warning'>[src] deactivated. Please, insert the power cell.</span>")
 			deactivate_sr()
 			return
 
-		if(cyborg.cell.charge < powercost * 2)
-			to_chat(cyborg, "<span class='warning'>Self-repair module deactivated. Please recharge.</span>")
+		if(cyborg.cell.charge < powercost * 20)
+			to_chat(cyborg, "<span class='warning'>Low power levels detected. [src] deactivated.</span>")
 			deactivate_sr()
 			return
 
 		if(cyborg.health < cyborg.maxHealth)
-			if(cyborg.health < 0)
-				repair_amount = -2.5
-				powercost = 30
-			else
-				repair_amount = -1
-				powercost = 10
-			cyborg.adjustBruteLoss(repair_amount)
-			cyborg.adjustFireLoss(repair_amount)
-			cyborg.updatehealth()
+			if(cyborg.health < cyborg.maxHealth / 2 && mode == "standard")
```

Comment:
Use defines instead of magic like "standard"

Reply:
Addressed in latest push 👍